nise/generators/gcp/gcp_generator.py

```diff
@@ -5,7 +5,7 @@
 
 from nise.generators.generator import AbstractGenerator
 
-
+# Old report columns
```

Comment:
I still need to delete the old columns then rename the `BIGQ_REPORT_COLUMNS` to `GCP_REPORT_COLUMNS`.

Reply:
addressed here: https://github.com/project-koku/nise/pull/314/commits/3e23ad4e34e94d146ab961688c6555c6203673ff